app/src/views/private/components/file-preview.vue

```diff
@@ -14,12 +14,15 @@
 			<v-icon-file :ext="type" />
 		</div>
 	</div>
+	<div v-else-if="imgError">
+		<v-error :error="imageError" />
+	</div>
```

Comment:
Does this need to be wrapped in another div?

```suggestion
	<v-error v-else-if="imgError" :error="imageError" />
```